formats/v1.9/summary_containers.json

```diff
@@ -172,7 +172,7 @@
                 "default": null,
                 "description": "The URL or xpath-like string pointing to the referred-to element",
                 "name": "href",
-                "required": false,
+                "required": true,
```

Comment:
This file of course isn't used by the autogenerator yet, but I see you're putting things in order for when it is.

Reply:
That's the general idea.